.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
IMHO let's do this weekly. Daily is far too much noise with dependabot.

Reply:
Fail enough. I addressed that comment with 8ebcdd5ba72a8e387b6bcbf4dc9f249d2a83e16d